### PR TITLE
Translator and Language Detector Tool using Python 3

### LINKED ISSUE #1128: Translator and Language Detector Tool using Python 3
# Description
This python script is to build a translator tool through which we can translate from one language to another, it can be applicable for a word, a sentence or a paragraph. Also this tool will detect the input language as well.

| name            | problem/motivation                                                  | possible solution  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Translator and Language Detector using Python 3 | Developing a Python script which will detect the input language and also it can translate the input text into a desired language as output | Yeah, I have the possible solution. I can build this using the python library called googletrans and pprint | self-assign @abhisheks008 |

**Screenshots/Proof of Concept**
[Translator Tool Raw Image 
](https://i.imgur.com/Q5a6coh.png)

This is the issue that I am going to build and fix. Please assign it to me.
Thanks and regards!

### PR DESCRIPTION
**Pull request by abhisheks008**
# Description

This python script is to build a translator tool through which we can translate from one language to another, it can be applicable for a word, a sentence or a paragraph. Also this tool will detect the input language as well

Fixes #1128 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:
- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by abhisheks008** [comment]
@HarshCasper Please take a look at this. I have informed this issue in the discord channel as well, why this is not being verified despite of the files are perfectly running and created successfully.

**Comment by abhisheks008** [comment]
- [x] LGMSOC Participant

**Comment by abhisheks008** [comment]
@HarshCasper Please check the PR

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The Code Quality is quite poor. Remove the Jupyter Notebook, sanitize the script and maybe then I would be able to review it.

**Comment by abhisheks008** [comment]
@HarshCasper updated the required changes.

File: Python/Translator_and_Language_Detector/Language Detection and Translation.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+from googletrans import Translator, constants
+from pprint import pprint
+
+
+translator = Translator()
+
+
+
+# translate a Indonesia text to english text (by default)
```
Comment (vybhav72954) [inline Python/Translator_and_Language_Detector/Language Detection and Translation.py:12]: Interesting, your profile seems Indian,  am I supposed to believe that Indonesian is your first language that's why you have made it default?

**Comment by vybhav72954** [comment]
Oh never mind, code plagiarised from [here](https://www.kaggle.com/khotijahs1/how-to-translate-and-detect-language-in-python).

Look I am open to code reuse (and trust me I am very lenient with it), but this is simply copy-pasting someone else's work. WOuld love to see some other contribution from you. Thanks

**Pull request closed without merging**